server/repositories/__tests__/offenderRepository.spec.js

```diff
@@ -127,13 +127,13 @@ describe('offenderRepository', () => {
     });
   });
 
-  describe('getEventsListForToday', () => {
-    it('calls the getEventsListForToday endpoint for a given ID', async () => {
+  describe('getCurrentEvents', () => {
+    it('calls the getCurrentEvents endpoint for a given ID', async () => {
       const client = {
         get: jest.fn().mockResolvedValue('SOME_RESULT'),
       };
       const repository = offenderRepository(client);
```

Comment:
I appreciate this is probably out of scope, but this is duplicated. Could be argued we can stick it in a common `beforeEach` block or abstract to a common setup function?